Userland/Utilities/unzip.cpp

```diff
@@ -20,16 +20,16 @@
 static bool unpack_zip_member(Archive::ZipMember zip_member, bool quiet)
 {
     if (zip_member.is_directory) {
-        if (mkdir(zip_member.name.characters(), 0755) < 0) {
+        if (Core::System::mkdir(zip_member.name, 0755).is_error()) {
             perror("mkdir");
```

Comment:
I don't think errno is reliable here, but we presumably want to print the returned error instead anyways.

```cpp
if (auto maybe_error = Core::System::mkdir(zip_member.name, 0755); maybe_error.is_error()) {
    warnln("Failed to create directory '{}': {}", zip_member.name, maybe_error.error());
```